<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoutingConfigDialog</class>
 <widget class="QDialog" name="RoutingConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routing Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="voiceNavigationCheckBox">
     <property name="text">
      <string>Audible Turn Instructions</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="soundRadioButton">
         <property name="text">
          <string>Play a sound</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="speakerRadioButton">
           <property name="text">
            <string>Use this speaker</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="speakerComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;a href=&quot;https://userbase.kde.org/Marble/CustomSpeakers&quot;&gt;Get more speakers&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RoutingConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RoutingConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceNavigationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>speakerComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceNavigationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>speakerRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceNavigationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speakerComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>speakerRadioButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
